src/components/FoodButton.js

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+function FoodButton({ fruitButtonCSS, imgPlusTitle, buttonDecrement, setFruit, fruit, buttonIncrement, notEnabled }) {
+
+    return (
+        <div className={fruitButtonCSS}>
+            <h2>{imgPlusTitle}</h2>
+            <button
+                onClick={() => setFruit(fruit - 1)}
```

Comment:
Als je de huidige state wil gebruiken, dan kun je beter die gebruiken die meegegeven wordt
```suggestion
                onClick={() => setFruit(curr => curr - 1)}
```
Hierdoor wordt er niet telkens een nieuwe functie aangemaakt in memory, omdat een waarde buiten de functie veranderd wordt